Libraries/src/Amazon.Lambda.KinesisEvents/KinesisTimeWindowResponse.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace Amazon.Lambda.KinesisEvents
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Runtime.Serialization;
+
```

Comment:
same comment about Amazon header and using statements.

Reply:
Thanks for pointing out. I just followed the convention in existing classes (copy/paste). None of the event classes have licensing header.